### PR TITLE
#9 feat: added build, clean and run shell scripts

### PR DESCRIPTION
**Pull request by darkokos**
Adds build, clean and run shell scripts, to ease plugin installation/uninstallation and running the server

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+pip install .\\file_system_to_graph_parser\\
```
Comment (teodord25) [inline build.sh:3]: what are these file separators

Comment (teodord25) [reply build.sh:3]: i dont like this has this been tesetd

Comment (teodord25) [reply build.sh:3]: windows style file separators in a bash script?

**Review by drag0dev — APPROVED** [review]

**Review by aleksaaaa02 — APPROVED** [review]
Looks great 👍

**Comment by teodord25** [comment]
image
@darkokos what did i tell u

**Comment by teodord25** [comment]
image

**Pull request merged**